### PR TITLE
With reference to issue #5666

### PR DESCRIPTION
**Pull request by Saanidhyavats**
- [ ] Tests added / passed
- [ ] Passes `black dask` / `flake8 dask`

### CODE REVIEW

File: dask/dataframe/core.py [hunk 1]
```diff
@@ -2252,7 +2252,7 @@ def astype(self, dtype):
         elif is_categorical_dtype(dtype) and getattr(dtype, "categories", None) is None:
             meta = clear_known_categories(meta)
         return self.map_partitions(
-            M.astype, dtype=dtype, meta=meta, enforce_metadata=False
+            M.astype, dtype=dtype, meta=meta, enforce_metadata=False, errors=errors
```
Comment (mrocklin) [inline dask/dataframe/core.py]: It looks like errors isn't defined anywhere.

**Comment by mrocklin** [comment]
Thank you for the pull request @Saanidhyavats .  it looks like this change would break things though.  Did you take a look at https://github.com/dask/dask/pull/5686 ?  That seems to be much further along.

**Comment by Saanidhyavats** [comment]
Thank you for guiding me @mrocklin . I will take a look at #5686 .

**Comment by TomAugspurger** [comment]
I'll note that it's not clear what the correct behavior is here. This
operation isn't easy to support lazily since the result metadata depends on
the values present. I think the best thing to do is to document that
`errors="ignore"` isn't supported.

**Comment by Saanidhyavats** [comment]
Shall I add errors=ignore isn't supported in document and remov the changes?

**Comment by TomAugspurger** [comment]
That's my preference.

**Comment by TomAugspurger** [comment]
@Saanidhyavats it looks like you pushed unrelated changes to this branch.

**Comment by Saanidhyavats** [comment]
Sorry for the inconvenience, I wanted to create a separate pull request for #3663 soon I will revert back the changes

**Comment by Saanidhyavats** [comment]
I am closing this pull request for now, if I get some solution for issue #5666 then I would contribute in pull request #5686 or I will make a new pull request.

**Pull request closed without merging**